cluster/gce/coreos/node.yaml

```diff
@@ -123,7 +123,7 @@ coreos:
         ExecStartPre=/usr/bin/mkdir -p /etc/kubernetes/manifests
         ExecStartPre=/run/setup-auth.sh
         ExecStart=/opt/kubernetes/server/bin/kubelet \
-        --api_servers=https://kubernetes-master.c.${PROJECT_ID}.internal \
+        --api_servers=https://${KUBERNETES_MASTER_NAME}.c.${PROJECT_ID}.internal \
```

Comment:
That is set to `KUBERNETES_MASTER_NAME` in the env file:

https://github.com/GoogleCloudPlatform/kubernetes/blob/b15e2e734029d73ee7685acaa1889aba896ff48f/cluster/gce/coreos/helper.sh#L76